libs/sdk-react-native/android/build.gradle

```diff
@@ -1,3 +1,12 @@
+import groovy.json.JsonSlurper
+
+def getVersionFromNpmPackage() {
+    def inputFile = new File("$rootDir/../node_modules/@breeztech/react-native-breez-sdk/package.json")
+    def packageJson = new JsonSlurper().parseText(inputFile.text)
+
+    return packageJson["version"]
+}
+
```

Comment:
I could imagine that at some point we'd like to have more flexibility and let the RN plugin version differ from the Android/iOS versions that it depends on. But for now I think it is fine to have them in sync.

Reply:
Does this mean that in order to set the react native version, you have to do a commit with the new version? Or does this take the version from here https://github.com/breez/breez-sdk/pull/407/files#diff-f44ac18980dab182be3760a350b5fc0a5218d4e93aee37d12d56d3bd8bdabf70R41 ?

Reply:
The correct way would be to set the version in the `package.json` and commit that.

But if you don't do that, the line you linked will overwrite whatever is specified in that file. 

TL;DR: it's going to use the version specified in the workflow run.